app/views/questions/question_content.html.erb

```diff
@@ -0,0 +1,249 @@
+            <h3>Заполните форму заявки и ждите звонка</h3>
+
+            <form method="get">
+
+              <div class="row">
+                <div class="large-4 columns">
+                  <input id="imea" type="text" value="" placeholder="Ф.И.О*">
+                  <input id="telefon" type="text" value="" placeholder="№ телефона*">
+                  <input id="email" type="text" value="" placeholder="email*">
+                  <input id="region" type="text" placeholder="регион*">
+                </div>
+                <div class="large-8 columns">
+                  <textarea id="vopros" value="" placeholder="задайте вопрос*"></textarea>
+                </div>
+              </div>
+
+              <div class="row">
+                <div class="large-6 columns">
+                  <span>Ваши данные не будут переданы третьим лицам.</span>
+                </div>
+                <div class="large-6 columns">
+                  <a href="#" class="button button_on_page form-button">отправить вопрос</a>
+                </div>
+              </div>
+            </form>
+          </div>
+        </div>
+      </div>
+      <div class="box-area">
+        <div class="row">
+          <div class="large-3 medium-3 small-3 columns">
+            <div class="box">
+              <a href="#" class="pen">
+                Заполните форму
+                заявки или закажите
+                обратный звонок.
+              </a>
+            </div>
+          </div>
+          <div class="large-3 medium-3 small-12 columns">
+            <div class="box">
+              <a href="#" class="girl">
+                Юрист перезвонит и
+                расскажет о этапах
+                решения проблемы.
+              </a>
+            </div>
+          </div>
+          <div class="large-3 medium-3 small-3 columns">
+            <div class="box">
+              <a href="#" class="list">
+                Поможем собрать
+                все необходимые
+                документы для дела.
+              </a>
+            </div>
+          </div>
+          <div class="large-3 medium-3 small-3 columns">
+            <div class="box">
+              <a href="#" class="flag">
+                Доведем дело
+                до логического конца.
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="comments-area">
+        <h4><%= @keyword.keyword %></h4>
+        <div class="comment-list">
+          <% @questions.each do |content| %>
+              <div class="comment">
+                <div class="ava">
+                  <img src="<%= raw content.user.avatar %>"/>
+                </div>
+                <div class="comment-body">
+                  <h4><%= content.user.firstname %> <%= content.user.lastname %></h4>
+                  <p><%= content.body %></p>
+                </div>
+                <div class="clearfix"></div>
+                <% if content.answers.present? %>
```

Comment:
like in the line question.answers.present? is more pleasant to read then content.answers.present?